setupext.py

```diff
@@ -30,6 +30,22 @@
 }
 """
 
+# Note: This code requires C++14 functionalities (also required to compile yt)
+# It compiles on gcc 4.7.4 (together with the entirety of yt) with the flag "-std=gnu++0x".
+# It does not compile on gcc 4.6.4 (neither does yt).
+CPPCODE = """
+#include <vector>
+
+struct node {
+    std::vector<int> vic;
+    bool visited = false;
+};
+
+int main() {
+    return 0;
+}
+"""
+
```

Comment:
Should this be moved to the one function that uses it ?

Reply:
I moved it together with another chunk of code that's checking for openmp support.